# Holloway_Czerwinski_Tsai_2020
This repository contains code for analyses as seen in "In vitro and in vivo development of the human intestinal niche at single cell resolution" - Emily M. Holloway, Michael Czerwinski, Yu-Hwai Tsai et. al (2020). 

Manuscript can be found here: https://www.biorxiv.org/content/10.1101/2020.01.31.928788v1

## Abstract

The human intestinal stem cell (ISC) niche supports ISC self-renewal and epithelial function, yet little is known about the development of the human ISC niche. We used single-cell mRNA sequencing (scRNA-seq) to interrogate the human intestine across 7-21 weeks of gestation. Using these data coupled with marker validation in situ, molecular identities and spatial locations were assigned to several cell populations that comprise the epithelial niche, and the cellular origins of many niche factors were determined. The major source of WNT and RSPONDIN ligands were ACTA2+ cells of the muscularis mucosa. EGF was predominantly expressed in the villus epithelium and the EGF-family member NEUREGULIN1 (NRG1) was expressed by subepithelial mesenchymal cells. Functional data from enteroid cultures showed that NRG1 improved cellular diversity, enhanced the stem cell gene signature, and increased enteroid forming efficiency, whereas EGF supported a secretory gene expression profile and stimulated rapid proliferation. This work highlights unappreciated complexities of intestinal EGF/ERBB signaling and identifies NRG1 as a stem cell niche factor.

## About

To run this project, install Scanpy from [here](https://github.com/theislab/scanpy). Make sure to have a recent version of Python. As of March 2020, we are using Python version 3.7.3.

To understand the single cell analysis tools, check out the Scanpy and Seurat tutorials side-by-side [here](https://scanpy-tutorials.readthedocs.io/en/latest/pbmc3k.html) and [here](https://satijalab.org/seurat/v3.0/pbmc3k_tutorial.html)

## Authors

* **Joshua Wu** wujos@med.umich.edu
* **Mike Czerwinski** czerwmj@med.umich.edu

Any questions, comments and/or feedback are welcome!

## Acknowledgments

* [**Jason Spence Lab**](http://www.jasonspencelab.com/) for support from our colleagues
* [**Theis Lab**](https://github.com/theislab) and [**Satija Lab**](https://satijalab.org/) for their work on modern single cell analysis techniques
